File: Java/length-of-last-word.java
/**
 * Problem: https://leetcode.com/problems/length-of-last-word/
 * 
 */
public class Solution {
    public int lengthOfLastWord(String s) {
        int len = 0;
        s = s.trim();
        for(int i=s.length() - 1; i>=0; i--){
            if(s.charAt(i) == ' ') {
                break;
            } else {
                len++;
            }
        }
        return len;
    }
}

File: Swift/length-of-last-word.swift
/**
 * Problem Link: https://leetcode.com/problems/length-of-last-word/
 *
 *
 *
 */

class Solution {
    func lengthOfLastWord(_ s: String) -> Int {
        var len = 0
        let rev = String(s.trimmingCharacters(in: .whitespacesAndNewlines).characters.reversed())
        for c in rev.characters {
            if String(c) == " " {
                return len
            } else {
                len += 1
            }
        }
        return len
    }
}
